docs/src/pages/[platform]/getting-started/usage/usage.flutter.mdx

```diff
@@ -1,7 +0,0 @@
-import { Alert, Text } from '@aws-amplify/ui-react';
-
-<Alert role="none" variation="info">
-  <Text>The Authenticator is the only supported Amplify UI component for Flutter.</Text>
```

Comment:
Rather than removing this page, should we add flutter to the supported frameworks in the link? I think it makes sense to point to the authenticator from this page since it is the only supported component in the Flutter framework.

https://ui.docs.amplify.aws/flutter/getting-started/usage

Reply:
The reason I removed it was because I thought this page basically doesn't have any informative content. Is that necessary since authenticator is the only thing accessible for Flutter on the side nav? 🤔

Reply:
yeah, it looks like it's also already mentioned in the install page. https://ui.docs.amplify.aws/flutter/getting-started/installation. I'm fine with deleting this.